### PR TITLE
build 2.3.0

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% set name = "tox" %}
-{% set version = "2.7.0" %}
-{% set sha256 = "9c3bdc06fe411d24015e8bbbab9d03dc5243a970154496aac13f9283682435f9" %}
+{% set version = "2.3.0" %}
```
Comment (jakirkham) [inline recipe/meta.yaml:2]: Why downgrade?

Comment (sannykr) [reply recipe/meta.yaml:2]: @jakirkham: I need two versions. First I built 2.7.0, and now I am building 2.3.0.

**Comment by jakirkham** [comment]
That's fine. Though this should be retargeted at a non-`master` branch (e.g. `2.3`). Also would double check the dependencies' version requirements.

**Comment by sannykr** [comment]
@jakirkham 
"retargeted at a non-master branch". Could you please explain it further. I forked the feedstock, and created a branch, made the required changes and then made a pull request. Is there anything else I need to do?

**Comment by jakirkham** [comment]
Toggling for CIs.

**Comment by jakirkham** [comment]
Have gone ahead and retargeted this PR to the `2.3` branch. Basically just created a branch on the main feedstock on an existing commit, which I called `2.3`. Then changed the base branch (where this PR will be merged into) from `master` to `2.3`. So this should be good to once it is passing.

**Comment by jakirkham** [comment]
I don't think there is any point in toggling for CIs at this point. It looks like there is an actual issue with the recipe and/or its dependencies. Until that is resolved these builds will continue to fail.

**Comment by jakirkham** [comment]
This should be `noarch: python` as well, which should speed up the testing and deployment.

**Comment by jakirkham** [comment]
Should add this requires a [re-render]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) as well.

**Comment by jakirkham** [comment]
To simplify things, let's try and get PR ( https://github.com/conda-forge/tox-feedstock/pull/4 ) in first and then we can rebase this on top of `master`.

**Comment by sannykr** [comment]
@jakirkham : When i tried to rerender, it says "No changes made".

**Comment by jakirkham** [comment]
Ok, please fetch the latest changes from this repo and then rebase your changes on top. If you are not sure how, please ask.

**Comment by jakirkham** [comment]
Rebased and squashed version in PR ( https://github.com/conda-forge/tox-feedstock/pull/5 ). Still gave you credit for the commit. Please let me know if that looks ok.

**Comment by jakirkham** [comment]
This can be closed now that PR ( https://github.com/conda-forge/tox-feedstock/pull/5 ) has been merged.

**Pull request closed without merging**